eng/tools/versioning/set-dev.js

```diff
@@ -47,23 +47,29 @@ async function commitChanges(repoPackages, pkg) {
   }
 }
 
-function updatePackageVersion(repoPackages, pkg, buildId, catalogs) {
+function updatePackageVersion(repoPackages, pkg, buildId, catalogs, logPrefix = "\t") {
   const currentVersion = repoPackages[pkg].json.version;
   const parsedVersion = semver.parse(currentVersion);
   repoPackages[pkg].newVer =
     `${parsedVersion.major}.${parsedVersion.minor}.${parsedVersion.patch}-alpha.${buildId}`;
-  console.log(`version updated for ${pkg}`);
+  console.log(`${logPrefix}version updated: from ${currentVersion} to ${repoPackages[pkg].newVer}`);
```

Comment:
Could we use https://developer.mozilla.org/en-US/docs/Web/API/console/group_static instead of passing the log prefix? This, by the way, is something I _just_ learned about 5 minutes ago 😄 

```
keyvault-keys main % node
Welcome to Node.js v20.18.3.
Type ".help" for more information.
> console.log("a");console.group();console.log("b");console.group();console.log("c");console.groupEnd();console.groupEnd();console.log("d")
a
  b
    c
d
undefined
> 
```

Reply:
updated